### DISCUSSION TITLE
Launch Template with Metadata (AWS::CloudFormation::Init)

### DISCUSSION THREAD

**Discussion by dshallenberger** [aws/aws-cdk#24773] [General]
Is there a better way to put metadata for AWS::CloudFormation::Init on a Launch Template? I found this old issue but there doesn't seem to be a fix. https://github.com/aws/aws-cdk/issues/13351

**Comment by isuftin** [comment]
@dshallenberger  - Did you ever find a solution? I've just started exploring working with CDK and the only thing that works right now is https://github.com/aws/aws-cdk/issues/13351#issuecomment-901006734
